apps/react-native/src/routes/plugin-permissions/plugin-permissions.controller.tsx

```diff
@@ -0,0 +1,52 @@
+// @ts-ignore
+import openURLInBrowser from 'react-native/Libraries/Core/Devtools/openURLInBrowser';
+
+import {Platform} from 'react-native';
+import {useNavigator} from '@brandingbrand/fsapp';
+import {PERMISSIONS, request} from 'react-native-permissions';
+
+export default (
+  props: PropsWithChildrenFunction<
+    {},
+    {
+      onPressDocs: () => void;
+      onPressClose: () => void;
+      onPressCamera: () => void;
+      onPressLocation: () => void;
+    }
+  >,
+) => {
+  const {children} = props;
+  const navigator = useNavigator();
+
+  const onPressDocs = () => {
+    openURLInBrowser(
+      'https://feat-flagship-12--whimsical-tartufo-49504f.netlify.app/en/packages/plugins/permissions',
```

Comment:
We should have this be in the runtime env so that we can easily change it. https://feat-flagship-12--whimsical-tartufo-49504f.netlify.app

Reply:
yesssss, good use case